### PR TITLE
Add Command+.lua

### PR DESCRIPTION
**Pull request by ghost**
I share my Command+.
Command+ it's a script for create news command for AC.

This script adding a few commands :
- player teleport #game_tele             === Teleport selected player to game_tele
- getmodel item | npc | gob #entry   === Return Model of #entry
- getnpc entry | guid | model             === Return Information of Selected NPC
- duplicate item | npc | gob #entry    === Duplicate *_template #entry on DB

In developpment : 
- player set faction #race                    === Set player faction for race enter

### CODE REVIEW

**Comment by Helias** [comment]
Nice!
I will test this in these days.

File: _submissions/lua_script/lua-command+.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Script info
+
+- Name: lua-command+
+- Author: iThorgrim-hub
+- Repository: https://github.com/iThorgrim-hub/Eluna-Catalogue/tree/lua-command+
+- Download: https://github.com/iThorgrim-hub/Eluna-Catalogue/archive/command+_v1.0.zip
```
Comment (Helias) [inline _submissions/lua_script/lua-command+.md:6]: A little suggestion: change the link to the release page.

https://github.com/iThorgrim-hub/Eluna-Catalogue/releases

Comment (ghost) [reply _submissions/lua_script/lua-command+.md:6]: No problem ;)

Comment (ghost) [reply _submissions/lua_script/lua-command+.md:6]: It' done

**Comment by P-Kito** [comment]
Question: You are doing Live DB Queries - Is it not possible to fetch it out of the existing containers?

**Comment by ghost** [comment]
Yes i's possible i make it ;)

**Comment by ghost** [comment]
It's now update :D With Request on start & Reload Server for Teleport :)

`
table.insert(CheatCommands.TpList, {name, map, position_x, position_y, position_z, orientation});
`

**Comment by Helias** [comment]
Hope to find the time today or tomorrow to test this and merge ;-)

P.S. = if you don't mind, use repo/**releases**/ as download link instead of the single release

**Comment by P-Kito** [comment]
Hmmm... it's not what I meant unfortunately :-X

I never did a lua extension script, but if I would do all of this in the core itself, I would NOT use "SELECT FROM", since there is already all the containers with all the information in the core.

So I would rather just fetch all the data from it:   selectedCreature->get...() and use that.

**Comment by ghost** [comment]
Yes I see.. I already do..
When you select a creature I ask Eluna to return the result via the functions that allow me to do so, but if you don't select a creature and you enter an entry, I ask Eluna to make a SQL query to get me some information about the entry, it is not possible to do otherwise for the moment ^^

**Comment by P-Kito** [comment]
Ahh. I thought Eluna was able to access the existing containers.
Well, then you need live-sql fetching...

**Comment by Helias** [comment]
Everything works! Cool!

immagine

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
Asap I will make a repository only for that lua_command+.lua script, to add it in the catalogue, because actually you have everything in a repo (?)

What should be the best solution to do that in your point of view?

**Comment by ghost** [comment]
We can do the same as with the rate exp modifier with the interface, we only send to this script on my repo as well as on the releases.

**Comment by BarbzYHOOL** [comment]
one repository for one release like for modules/tools, that is the best way and the most manageable

it's never good to compile everything into one big repo

**Comment by ghost** [comment]
Ok, everything is avaible here : https://github.com/iThorgrim-hub/lua-command-plus

**Comment by Helias** [comment]
Create an icon if you can :P, anyway, forked!
I added you as mantainer of the repo, hope you can keep the updates in the AC repo
Next time we can avoid the fork and just create the lua scirpt in a new AC repo

**Comment by BarbzYHOOL** [comment]
Ithorgrim has his own repo, we can use his repo as a base?

**Pull request merged**